app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -269,9 +269,12 @@ public void onBackPressed() {
                 finish();
             }
         } else if (getSupportFragmentManager().findFragmentByTag(nearbyFragmentTag) != null && !isContributionsFragmentVisible) {
-            // Meas that nearby fragment is visible (not contributions fragment)
+           if (!((NearbyFragment)contributionsActivityPagerAdapter.getItem(1)).listOptionMenuIteClicked()){
+               viewPager.setCurrentItem(0);
+           }
+                //
```

Comment:
Can we remove this dangling comment, and maybe add some more comments explaining what this does e.g.

"The nearby fragment is visible"
"The nearby bottom sheet is visible"